__tests__/src/services/urlBuilder.spec.js

```diff
@@ -0,0 +1,51 @@
+import * as UrlBuilder from '../../../src/services/urlBuilder';
+
+describe('urlBuilder', () => {
+    describe('addGetParams', () => {
+        test('should not change the url when there is no params', async () => {
+            const url = 'http://my.url';
+
+            const expectedUrl = 'http://my.url';
+            const result = UrlBuilder.addGetParams(url);
+
+            expect(result).toEqual(expectedUrl);
+        });
+
+        test('should add one param to the url', async () => {
+            const url = 'http://my.url';
+            const params = {
+                param1: 'test',
+            };
+
+            const expectedUrl = 'http://my.url?param1=test';
+            const result = UrlBuilder.addGetParams(url, params);
+
+            expect(result).toEqual(expectedUrl);
+        });
+
+        test('should add multiple params to the url', async () => {
+            const url = 'http://my.url';
+            const params = {
+                param1: 'test',
+                param2: JSON.stringify([[1, 2, 3], [4, 5, 6], [7, 8, 0]]),
+            };
+
+            const expectedUrl =
+                'http://my.url?param1=test&param2=[[1,2,3],[4,5,6],[7,8,0]]';
```

Comment:
What's `¶m2=` ?

Reply:
Forget that, only github displaying  weird character on the "Files changed" view.